packages/web-react/src/types/shared/style.ts

```diff
@@ -32,6 +33,8 @@ export interface StyleProps extends SpacingProps {
   UNSAFE_style?: CSSProperties;
 }
 
+export type TextAlignmentProps = AlignmentXDictionaryType | Record<BreakpointToken, AlignmentXDictionaryType>;
```

Comment:
```suggestion
export type TextAlignmentType = AlignmentXDictionaryType | Record<BreakpointToken, AlignmentXDictionaryType>;
```

This should be probably named Type instead of Props 🤷‍♂️